nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-processor-configuration.js

```diff
@@ -264,6 +251,7 @@ nf.ProcessorConfiguration = (function () {
             processorConfigDto['schedulingPeriod'] = schedulingPeriod.val();
         }
 
+        processorConfigDto['executionNode'] = $('#execution-node-combo').combo('getSelectedOption').value;
```

Comment:
Should probably consider whether the execution node is available before including its value in the processor's configuration.